webhook/src/middleware/auth.js

```
@@ -31,12 +40,21 @@ const authenticateFirebase = async (req, res, next) => {
     }
     req.user = {
       uid: req.headers['x-test-uid'] || 'test-user-id',
+      email: req.headers['x-test-email'] || 'test@example.com',
```

Biome (2.4.13)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)